code/game/objects/effects/decals/contraband.dm

```diff
@@ -362,6 +365,36 @@
 	desc = "The Griffin commands you to be the worst you can be. Will you?"
 	icon_state = "poster33"
 
+/obj/structure/sign/poster/contraband/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/toy/crayon/red/syndicate))
+		to_chat(user, "<span class='notice'>There is no reason to change the message of this poster.</span>")
+		return
+	else
+		..()
+
+/obj/structure/sign/poster
+	var/corrupted_icon
+	var/corrupted_desc
+	var/corrupted_name
+
+/obj/structure/sign/poster/official/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/toy/crayon/red/syndicate))
+		var/obj/item/toy/crayon/red/syndicate/C = I
+		user.visible_message("<span class='notice'>[user] starts drawing over [src].</span>", "<span class = 'notice'>You start drawing over [src] with your crayon.</span>")
+		if(do_after(user, 50, target = src))
+			C.uses -= 5
+			user.visible_message("<span class='notice'>[user] finishes drawing over [src].</span>", "<span class = 'notice'>You finish drawing over [src] with your crayon.</span>")
```

Comment:
spacing issue still here in the self message. Same above.
You can also make use of C here to use it's name and such.